src/plugins/helper.js

```diff
@@ -97,7 +97,12 @@ const update = (parent, reference, entry) => {
           }
         }
       } else {
-        entry = entry[parent[j]]
+        const key = _parent[j];
```

Comment:
@cs-raj , QA required for these changes. It's better to only merge licence fix. Code changes @AniketDev7 need to do the QA.
CC: @netrajpatel

Reply:
Hi @aman19K the changes have been reverted